common/types/metrics.ts

```diff
@@ -13,10 +14,15 @@ export interface MetricType extends VisualizationType {
   w: number;
   h: number;
   query: {
-    type: 'savedCustomMetric' | 'prometheusMetric';
+    type: 'savedCustomMetric' | 'prometheusMetric' | typeof OTEL_METRIC_SUBTYPE;
```

Comment:
nit. Can we move all the three in a single type

Reply:
moved it, will update now

Reply:
changed it